File: 240105/문자열 길이의 합/sum-length-of-string.py
n = int(input())
word = []
val =  0
tal = 0 
for i in range(n):
    word.append(input())
    tal = len(word[i]) + tal

for j in range(n):

    for k in range(len(word[j])):
        if word[j][k] =='a':
            val+=1
print(tal,val)

File: 240105/문자열 길이의 합/sum-length-of-string.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        String[] word = new String[n];
        int tal = 0 ;
        int val = 0 ;
        for (int i = 0 ; i<n; i++){
            word[i] = sc.next();
            tal +=word[i].length();
            if(word[i].charAt(0)=='a'){
                val+=1;
            }
        }
        System.out.print(tal+" "+val);
    }
}